File: dto/Booking.java
package com.cg.cabbookingsystem.dto;

public class Booking {

	private int bookingId;
	private String source;
	private String destination;
	private int vehicleSize;
	private int driverId;
	private int customerId;
	private String driverStatus;
	private String customerStatus;
	private String tripStatus;
	private double estimatedFare;
	private double finalFare;
	private int rating;
	
	public int getBookingId() {
		return bookingId;
	}
	public void setBookingId(int bookingId) {
		this.bookingId = bookingId;
	}
	public String getSource() {
		return source;
	}
	public void setSource(String source) {
		this.source = source;
	}
	public String getDestination() {
		return destination;
	}
	public void setDestination(String destination) {
		this.destination = destination;
	}
	public int getVehicleSize() {
		return vehicleSize;
	}
	public void setVehicleSize(int vehicleSize) {
		this.vehicleSize = vehicleSize;
	}
	public int getDriverId() {
		return driverId;
	}
	public void setDriverId(int driverId) {
		this.driverId = driverId;
	}
	public String getDriverStatus() {
		return driverStatus;
	}
	public void setDriverStatus(String driverStatus) {
		this.driverStatus = driverStatus;
	}
	public String getCustomerStatus() {
		return customerStatus;
	}
	public void setCustomerStatus(String customerStatus) {
		this.customerStatus = customerStatus;
	}
	public String getTripStatus() {
		return tripStatus;
	}
	public void setTripStatus(String tripStatus) {
		this.tripStatus = tripStatus;
	}
	
	
}

File: Integrated-Angular/transit-app/src/app/model/booking.ts
export class Booking{
    constructor (public bookingId: number=null,
        public customerId: number=null,
        public driverId: number=null,
        public source: string="",
        public destination: string="",
    //    public vehicleSize: number=null,
        public driverStatus:string="",
    //    public customerStatus:string="",
        public tripStatus:string="",
        public estimatedFare:number=null,
        public estimatedTime:number=null,
        public finalFare:number=null,
        public rating:number=null, 
        public issues:string="", 
        public finalTime:number=null
        ){}
}